Tests/BugsnagTests/PayloadTransformerTests.swift

```diff
@@ -84,7 +93,11 @@ class PayloadTransformerTests: XCTestCase {
     }
 
     func testThatSeverityIsCorrect() {
-        XCTAssertEqual(payload["events"]?[0]?["severity"]?.string, "error")
+        XCTAssertEqual(payload["events"]?[0]?["severity"]?.string, self.severity.rawValue)
+    }
+
+    func testThatSeverityGetsDefaultValue() {
```

Comment:
Consider moving the `payloadDefault` down into this test as it's only used here.